packages/react-reconciler/src/ReactFiberClassComponent.js

```diff
@@ -364,6 +366,22 @@ export default function(
         name,
         name,
       );
+
+      if (
+        typeof instance.getSnapshotBeforeUpdate === 'function' &&
+        typeof instance.componentDidUpdate !== 'function' &&
+        typeof instance.UNSAFE_componentDidUpdate !== 'function' &&
```

Comment:
What is `UNSAFE_componentDidUpdate`?

Reply:
Same as above. Just trying to cover more cases.

Reply:
To be clear I mean that there's no lifecycle called `UNSAFE_componentDidUpdate` at all. It's not an actual alias we use because `componentDidUpdate` is not deprecated.

Reply:
😮 Yeah, duh. I was reading "did" and thinking "will".

Reply:
Sorry. I misinterpreted this comment to be another like the one in the test above. This is a legit mistake.